src/tree/FunctionTreeItem.ts

```diff
@@ -49,8 +52,11 @@ export class FunctionTreeItem implements IAzureTreeItem {
     public async deleteTreeItem(node: IAzureNode<FunctionTreeItem>): Promise<void> {
         const message: string = localize('azFunc.ConfirmDelete', 'Are you sure you want to delete "{0}"?', this._name);
         if (await vscode.window.showWarningMessage(message, DialogResponses.yes, DialogResponses.cancel) === DialogResponses.yes) {
+            this._outputChannel.show(true);
+            this._outputChannel.appendLine(localize('DeletingFunction', 'Deleting function "{0}"...', this._name));
```

Comment:
Is this worth moving to the shared npm package?  The functionality is the same, we would just need to check when to use client.functionModel and client.webApps, right?

Reply:
IMO - not worth it. The code isn't that complex and it is slightly different for functions vs. sites.

We could move it if other extensions ever need to delete a function.